grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandler.java

```diff
@@ -157,6 +152,14 @@ static UnframedGrpcErrorHandler ofPlainText(UnframedGrpcStatusMappingFunction st
                                                                    .addInt(GrpcHeaderNames.GRPC_STATUS,
                                                                            grpcCode.value())
                                                                    .build();
+            final StringBuilder message = new StringBuilder("grpc-code: " + grpcCode.name());
+            final String grpcMessage = status.getDescription();
+            if (grpcMessage != null) {
+                message.append(", ").append(grpcMessage);
+            }
+            if (cause != null && ctx.config().verboseResponses() && !status.isOk()) {
```

Comment:
nit; I believe this whole code block is triggered if `status != OK`.

https://github.com/line/armeria/blob/088202fea3ceab2408da2eaa1f763242135e76ea/grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractUnframedGrpcService.java#L180

```suggestion
            if (cause != null && ctx.config().verboseResponses()) {
```

Reply:
Ah, I added it for just safe. If you think it is redundant, let me remove it. 🙇